spec/ValidationAndPasswordsReset.spec.js

```diff
@@ -527,6 +527,39 @@ describe('Custom Pages, Email Verification, Password Reset', () => {
       });
   });
 
+  it('succeeds sending a password reset email with case insensitive email', done => {
+    reconfigureServer({
+      appName: 'coolapp',
+      publicServerURL: 'http://localhost:1337/1',
+      emailAdapter: MockEmailAdapterWithOptions({
+        fromAddress: 'parse@example.com',
+        apiKey: 'k',
+        domain: 'd',
+      }),
+    })
+      .then(() => {
+        const user = new Parse.User();
+        user.setPassword('asdf');
+        user.setUsername('zxcv');
+        user.set('email', 'testCaseInsensitive@parse.com');
+        user
+          .signUp(null)
+          .then(() => Parse.User.requestPasswordReset('TESTCASEINSENSITIVE@parse.com'))
+          .then(
+            () => {
+              done();
+            },
+            error => {
+              done(error);
+            }
+          );
+      })
+      .catch(error => {
+        fail(JSON.stringify(error));
```

Comment:
Not necessary, it's just a test case so I'd say keep it simple and just fail.

Reply:
Changed.